### PR TITLE
[NG] Added color legend in Members page

### LINKED ISSUE #1071: [NG] Add color legend in members page
**Desired Behavior**
Add a color legend on the left side of `Active Members`(green) and `Inactive Members`(gray) in the dropdown present in the [community members page](https://layer5ng.netlify.app/community/members). The CSS for the color boxes should already be present. The component we are using for drop-down is [react-select](https://react-select.com/)

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by EmperorYP7**
**Description**

This PR fixes #1071

**Notes for Reviewers**

- Used CSS to make color legends for Member's page

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

Screenshot 2021-02-07 at 5 14 32 PM

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
I wonder, isn't it possible to do it without using any library and only using CSS?

**Comment by EmperorYP7** [comment]
@Nikhil-Ladha I was thinking the same! But, I thought to give it a shot when I was react-select's docs.. 😓 Anyways, I'll do it with CSS now.

**Comment by leecalcote** [comment]
Quick check... This is still in draft, right, @EmperorYP7?

**Comment by EmperorYP7** [comment]
@leecalcote Yep. Working on it.. 🔨

**Review by Jashpatel1 — CHANGES_REQUESTED** [review]
@EmperorYP7  The dropdown goes behind the member card and makes it inaccessible.

File: src/sections/Community/Members-grid/index.js [hunk 1]
```diff
@@ -32,6 +74,7 @@ const MembersGrid = props => {
                   value={props.members}
                   onChange={props.handleChange}
                   theme={dropdownTheme}
+                  styles={colourStyles}
```
Comment (EmperorYP7) [inline src/sections/Community/Members-grid/index.js:77]: @Jashpatel1 Even if I pass `colourStyles = {}` to `styles` prop in the react-select component, its behaving that way. Not sure why.

Comment (EmperorYP7) [reply src/sections/Community/Members-grid/index.js]: Never mind. 😓 fixed it!

**Comment by EmperorYP7** [comment]
@Jashpatel1 I think now it is fixed.

File: src/pages/community/members.js [hunk 2]
```diff
@@ -14,11 +14,11 @@ import Maintainers from "../../sections/Community/Members-grid/Maintainers";
 import Meshmate from "../../sections/Community/Members-grid/Meshmate";
 
 const options = [
-  { label: "All Members", value: "all" },
-  { label: "Active Members", value: "active" },
-  { label: "Inactive Members", value: "inactive" },
-  { label: "Maintainers", value: "maintainers" },
-  { label: "MeshMates", value: "meshmates" }
+  { label: "All Members", value: "all", color: "#0bb39f", isFixed: true},
```
Comment (leecalcote) [inline src/pages/community/members.js:17]: It'd be great if we could use the properties from src/theme/app/themeStyles.js here.

**Comment by leecalcote** [comment]
@EmperorYP7 good work so far. 👍 With a custom component comes a custom user experience. Reflecting on that for a moment, you might consider that a user would internalize the gray "Inactive Members" as a disabled option in the otherwise list of green-colored selections. You almost might find that users are confused when seeing a green circle next to Active Members AND next to All Members. A split-colored indicator using both green and gray might more accurately depict the fact that All Members is not filtered at all. In fact, not using a color indicator for this selection is best. Will you consider the following changes?

1. All selections in black text.
2. No status indicator for All Members.
3. Use of the "5" icon for the status indicator for Maintainers.
4. Use of the MeshMate icon for the status indicator for MeshMates.
5. Display of the status indicators in the dropdown list in addition to the status indicator shown after selection.

Screen Shot 2021-02-05 at 2 40 26 PM

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Marking with Post-go-live milestone.

**Comment by EmperorYP7** [comment]
@leecalcote On it! 🔨

**Comment by leecalcote** [comment]
Pretty cool to see this coming along, @EmperorYP7. 👍

**Comment by EmperorYP7** [comment]
> Will you consider the following changes?
> 
> 1. All selections in black text.
> 2. No status indicator for All Members.
> 3. Use of the "5" icon for the status indicator for Maintainers.
> 4. Use of the MeshMate icon for the status indicator for MeshMates.
> 5. Display of the status indicators in the dropdown list in addition to the status indicator shown after selection.
> 6. Use green indicator for active members.
> 7. Use gray indicator for inactive members.

@leecalcote Done! 🙆🏻‍♂️

File: src/sections/Community/Members-grid/membersGrid.style.js [hunk 3]
```diff
@@ -1,8 +1,100 @@
...
+});
 
 export const selectStyles = {
-  menu: styles => ({ ...styles, zIndex: 999, })
+  menu: styles => ({ ...styles, zIndex: 999, }),
+  control: styles => ({ ...styles, backgroundColor: "white", zIndex: 900 }),
+  option: (styles, { data, isFocused, isSelected }) => ({
+    ...styles,
+
+    alignItems: "center",
```
Comment (EmperorYP7) [inline src/sections/Community/Members-grid/membersGrid.style.js:47]: This is where I was sceptical about going against DRY. Any clever work-around for this?

**Review by EmperorYP7 — COMMENTED** [review]
I think the quality of my patch can be a lot better. All suggestions are welcomed! :smile:

**Review by Jashpatel1 — APPROVED** [review]
Looks good. For now, we can go ahead with this approach. Great work here. Thanks @EmperorYP7

**Review by leecalcote — APPROVED** [review]
Ditto. Good work. 😀

**Pull request merged**